### PR TITLE
Use new property for exhibition images (AIC-612)

### PR DESCRIPTION
**Pull request by sam33rdhakal**
* Adds the new attribute `image_url` to `ArticExhibition`
* Updates the view logic so that imageView's size is used to build the image URL 
`url=baseurl?w=imageView.measuredWidth&h=imageView.measuredHeight`

### CODE REVIEW

File: db/src/main/kotlin/edu/artic/db/models/ArticExhibition.kt [hunk 1]
```diff
@@ -44,6 +44,11 @@ data class ArticExhibition(
             return aic_start_at.toCurrentTimeZone()
         }
 
+    val thumbUrl: String?
+        get() {
+            return "$imageUrl?w=200&h=150"
```
Comment (Cliabhach) [inline db/src/main/kotlin/edu/artic/db/models/ArticExhibition.kt]: If `imageUrl` is nullable, should we maybe return null or an empty string if `imageUrl` is null?

Comment (sam33rdhakal) [reply db/src/main/kotlin/edu/artic/db/models/ArticExhibition.kt]: Yea. How did i miss that. Will update it. Thanks.

Comment (sam33rdhakal) [reply db/src/main/kotlin/edu/artic/db/models/ArticExhibition.kt]: Updated.

File: content_listing/src/main/kotlin/edu/artic/exhibitions/AllExhibitionsAdapter.kt [hunk 2]
```diff
@@ -17,13 +18,18 @@ import kotlinx.android.synthetic.main.cell_all_exhibitions_layout.view.*
...
 
     override fun View.onBindView(item: AllExhibitionsCellViewModel, position: Int) {
-        item.exhibitionImageUrl.subscribe {
-            GlideApp.with(context)
-                    .load(it)
-                    .error(R.drawable.placeholder_medium_square)
-                    .placeholder(R.color.placeholderBackground)
-                    .into(image)
-        }.disposedBy(item.viewDisposeBag)
+        item.exhibitionImageUrl
```
Comment (Cliabhach) [inline content_listing/src/main/kotlin/edu/artic/exhibitions/AllExhibitionsAdapter.kt]: Do we have to worry about `exhibitionImageUrl` being empty here?

Comment (sam33rdhakal) [reply content_listing/src/main/kotlin/edu/artic/exhibitions/AllExhibitionsAdapter.kt]: If its empty we'll display the placeholder image. So we don't need to worry.

**Pull request merged**